src/styles/sign-up-page.scss

```diff
@@ -385,6 +385,29 @@ $red-button: #D0021B;
     transform: translate(-50%, -50%);
   }
 
+  .marker-point__new {
+    background-color: rgba(255, 255, 255, 0.7);
```

Comment:
put colors in constants file you can then do: `rgba($white, .7)`. If you think that you are going to use this color with that alpha often you should put the color with the alpha in the constants file. Same for the other hardcoded colors in the file.
There is also no need for the 0 when writing decimals just `.7` will do.